plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -357,19 +357,20 @@ public void testAnalyzeSomeColumns()
                         "('name', null, null, 0.0, null, null, null)," +
                         "(null, null, null, null, 50.0, null, null)");
 
+        String expectedFullStats = "VALUES " +
+                "('nationkey', null, 50.0, 0.0, null, 0, 49)," +
+                "('regionkey', null, 10.0, 0.0, null, 0, 9)," +
+                "('comment', 3764.0, 50.0, 0.0, null, null, null)," +
+                "('name', 379.0, 50.0, 0.0, null, null, null)," +
+                "(null, null, null, null, 50.0, null, null)";
+        assertUpdate(format("ANALYZE %s WITH(recalculate_all_statistics = true)", tableName));
```

Comment:
While testing, I tried the combination

```
ANALYZE %s WITH(recalculate_all_statistics = true, columns = ARRAY['nationkey', 'regionkey']
```

Is it a bit confusing to you as well to use "recalculate_all_statistics" , but actually do it only for a specific set of columns?
Just thinking out loud whether the name chosen for the new `ANALYZE` property is sane.

Reply:
ok i will try to rename